### ISSUE TITLE
Pop_OS! 21.04 Installer fails to check for sufficient space before installing

### ISSUE THREAD

**Issue by MrChromebox** [pop-os/beta#161]
**Issue/Bug Description:**

Pop_OS! 21.04 beta fails to install due to insufficient space; doesn't check before install, doesn't report reason for installation failure. syslog reports:

pop-os io.elementary.installer.desktop[2380]: [WARN distinst:crates/chroot/src/command.rs:98] E: Write error - write (28: No space left on device)

**Steps to reproduce:**

Attempt to install Pop_OS 21.04 beta (clean) on a ChromeOS device with 16GB internal storage

**Expected behavior:**

Pop_OS! installer refuses to install due to insufficient space on the drive

**Comment by mmstick** [comment]
There is an insufficient space check, but perhaps its underestimating here. Maybe it's best to ditch the recovery partition with such a small amount of storage. I assume this was an EFI install.

**Comment by MrChromebox** [comment]
yes, this was an EFI install. Installer created a ~4GB FAT32 partition (recovery?) in addition to the EFS partition, along with a ~4GB swap partition, leaving only ~6.3GB for the OS

**Comment by mmstick** [comment]
Yeah, we can do without swap as well. Anyone needing swap can create a swapfile
